sormas-backend/src/main/java/de/symeda/sormas/backend/report/AggregateReportService.java

```diff
@@ -83,8 +84,10 @@ public Predicate createCriteriaFilter(
 	public Predicate createUserFilter(CriteriaBuilder cb, CriteriaQuery cq, From<AggregateReport, AggregateReport> from) {
 
 		User currentUser = getCurrentUser();
+		final JurisdictionLevel jurisdictionLevel = UserRole.getJurisdictionLevel(currentUser.getUserRoles());
 		if (currentUser == null
-			|| currentUser.hasAnyUserRole(UserRole.NATIONAL_USER, UserRole.NATIONAL_CLINICIAN, UserRole.NATIONAL_OBSERVER, UserRole.REST_USER)) {
+			|| (jurisdictionLevel == JurisdictionLevel.NATION && !currentUser.hasAnyUserRole(UserRole.POE_NATIONAL_USER))
```

Comment:
For the POE check I'd use UserRole.isPortHealthUser

Reply:
here it's only about ignoring the poe national user

Reply:
True, but from the logic side it's sufficient to check POE in general and the goal is to use as many explicit user roles as possible.